packages/frontend/src/hooks/useOrganizationGroups.ts

```diff
@@ -18,3 +19,35 @@ export const useOrganizationGroups = () => {
         onError: (result) => setErrorResponse(result),
     });
 };
+
+const createGroupQuery = async (data: { name: string }) =>
```

Comment:
not a blocker, but the type `{name: string}` could be added to the `ApiResults` in common so it can reused here and also on `useMutation<Group, ApiError, { name: string }>(`

Reply:
Yea, i should have said, but I think we will redo these types a bit so I didn't make them very fleshed out. If it's cool with you, I am going to skip this for now and add shared types in a follow up.